scripts/build/build_musllinux_1_1.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This script is more or less a copy of `scripts/build/build_manylinux_2_24.sh` some minor tweaks:

```diff
diff --git a/scripts/build/build_manylinux_2_24.sh b/scripts/build/build_musllinux_1_1.sh
index 6a9e992b..90ad9cca 100755
--- a/scripts/build/build_manylinux_2_24.sh
+++ b/scripts/build/build_musllinux_1_1.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Create manylinux_2_24 wheels for psycopg2
+# Create musllinux_1_1 wheels for psycopg2
 #
 # Look at the .github/workflows/packages.yml file for hints about how to use it.
 
@@ -27,16 +27,13 @@ if [[ "${PACKAGE_NAME:-}" ]]; then
 fi
 
 # Install prerequisite libraries
-curl -k -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-echo "deb http://apt.postgresql.org/pub/repos/apt stretch-pgdg main" \
-    > /etc/apt/sources.list.d/pgdg.list
-apt-get -y update
-apt-get install -y libpq-dev
+apk update
+apk add postgresql-dev tzdata
 
 # Create the wheel packages
 for pyver in $PYVERS; do
     pybin="/opt/python/${pyver}/bin"
-    "${pybin}/pip" wheel "${prjdir}" -w "${prjdir}/dist/"
+    "${pybin}/python" -m build -w -o "${prjdir}/dist/" "${prjdir}"
 done
 
 # Bundle external shared libraries into the wheels
@@ -44,12 +41,6 @@ for whl in "${prjdir}"/dist/*.whl; do
     auditwheel repair "$whl" -w "$distdir"
 done
 
-# Make sure the libpq is not in the system
-for f in $(find /usr/lib /usr/lib64 -name libpq\*) ; do
-    mkdir -pv "/libpqbak/$(dirname $f)"
-    mv -v "$f" "/libpqbak/$(dirname $f)"
-done
-
 # Install packages and test
 cd "${prjdir}"
 for pyver in $PYVERS; do
@@ -68,8 +59,3 @@ for pyver in $PYVERS; do
 
     "${pybin}/python" -c "import tests; tests.unittest.main(defaultTest='tests.test_suite')"
 done
-
-# Restore the libpq packages
-for f in $(cd /libpqbak/ && find . -not -type d); do
-    mv -v "/libpqbak/$f" "/$f"
-done
```

Reply:
```diff
-# Make sure the libpq is not in the system
```
Can we restore these lines? They check that the binary packages work even without the system libpq and I'd like to keep that check.